### PR TITLE
[CLB-37] Implement Header Dropdown Menu

### PR DESCRIPTION
**Pull request by Zola987**
Add Header dropdown menu with Logout funcionality.

### CODE REVIEW

File: src/components/DropdownMenu/DropdownMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type Props = {
+    setOpenMenu: (openMenu: boolean) => void;
+};
+
+const DropdownMenu: React.FC<Props> = ({ setOpenMenu }) => {
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+
+    const logout = () => {
```
Comment (Mbotunjac) [inline src/components/DropdownMenu/DropdownMenu.tsx:16]: Far as I can see it all looks god, just wondering is it better practice if this is in return. Found it somewhere so just a thought. 
`const { t } = useTranslation();` on a line 27.

**Comment by kimnovak** [comment]
There should be some margins between the icon and label
image

Also, can you make the dropdown arrow rotate when the dropdown is toggled? And if you can animate the rotation that would be awesome

File: src/components/DropdownMenu/DropdownMenu.tsx (hunk 1 above)
Comment (Zola987) [reply src/components/DropdownMenu/DropdownMenu.tsx:16]: You cannot place hooks in return.

Comment (Mbotunjac) [reply src/components/DropdownMenu/DropdownMenu.tsx:16]: you are right, I actually switched it in my head with render

**Review by bojanab987 — APPROVED** [review]

File: src/components/Header/header.module.css [hunk 2]
```diff
@@ -33,6 +33,16 @@
     border-radius: 25px;
 }
 .faCaretDown {
+    transform: rotate(0deg);
+    transition: transform 1s linear;
```
Comment (kimnovak) [inline src/components/Header/header.module.css]:
```suggestion
    transition: transform .3s linear;
```

File: src/components/Header/header.module.css [hunk 3]
```diff
@@ -33,6 +33,16 @@
...
+    cursor: pointer;
+    font-size: 28px;
+    color: var(--text-primary-color);
+}
+
+.open {
+    transform: rotate(180deg);
     font-size: 28px;
+    transition: transform 1s linear;
```
Comment (kimnovak) [inline src/components/Header/header.module.css]:
```suggestion
    transition: transform .3s linear;
```

**Pull request merged**